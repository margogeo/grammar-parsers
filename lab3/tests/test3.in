a = f(1, 2, 3)
b = f(1, -2, 3)
c__8908 = f(1, 2, -(3))
