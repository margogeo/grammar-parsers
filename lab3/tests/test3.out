O0OI0 = f(1, 2, 3)
IO1000O1=6887*3717
OIOI0 = f(1, -2, 3)
O1OI011001100OI10I1OO1OI1 = f(1, 2, -(3))

I01O00O10=658*II0O0IO1III0
